graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -89,18 +90,16 @@ def get_superclasses_from_class_definition(class_definition):
 
 
 class SchemaElement(object):
-    ELEMENT_KIND_VERTEX = 'vertex'
-    ELEMENT_KIND_EDGE = 'edge'
-    ELEMENT_KIND_NON_GRAPH = 'non-graph'
+    # pylint: disable=metaclass-assignment
+    __metaclass__ = ABCMeta
```

Comment:
I assigned the metaclass to ABCMeta in order to make the class an abstract class in a python2/python3 compatible way.